src/argilla/training/openai.py

```diff
@@ -56,7 +57,13 @@ def __init__(self, *args, **kwargs):
             self._logger.warning("Seed is not supported for OpenAI. Ignoring seed for training.")
 
         if self._model is None:
-            self._model = "curie"
+            self._model = "gpt-3.5-turbo"
+
+        if any([self._model in model for model in OPENAI_LEGACY_MODEL]):
```

Comment:
```suggestion
        if self._model in OPENAI_LEGACY_MODEL:
```

Reply:
This isn't correct behaviour because there can also be models like "babage-002" etc.